notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>(Same comment and solution as in the previous cell.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>